jobs/cgroup2.yaml

```diff
@@ -1,3 +1,4 @@
+testcase: cgroup2
 cgroup2:
     memory.high: 90%
     memory.low: 50%
```

Comment:
thanks, do you want to run it as a standalone test, or what is the use case?

So far, this is more like a reusable yaml to be included by others.
`jobs/pigz-cgroup2.yaml:1:<< : jobs/cgroup2.yaml
jobs/will-it-scale-cgroup2.yaml:1:<< : jobs/cgroup2.yaml
jobs/kbuild-cgroup2.yaml:1:<< : jobs/cgroup2.yaml
jobs/unixbench-cgroup2.yaml:1:<< : jobs/cgroup2.yaml
jobs/hackbench-cgroup2.yaml:1:<< : jobs/cgroup2.yaml
jobs/netperf-cgroup2.yaml:1:<< : jobs/cgroup2.yaml
jobs/ebizzy-cgroup2.yaml:1:<< : jobs/cgroup2.yaml
`

Reply:
yes... I run it as a standalone testcase. Will this line affect other testcases which include this cgroup2 base yaml?

Reply:
probably i guess, depends on the place it is included, like before or after another testcase define.

If you want to have a standalone test, you can consider add a new job like cgroup2-test.yaml with cgroup2-test as test case.

Reply:
This is more reasonable. I will have a try and update this pr. Thanks!